src/routes/responses/responses.controller.js

```diff
@@ -0,0 +1,53 @@
+const _get = require('lodash/get');
+const db = require("./../../db/index");
+
+// Organize Response Object from Slack API
+const postResponses = function (req, res) {
+  const body = JSON.parse(req.body.payload);
+  const response = {
+    responseSelected: _get(body, 'actions[0].text.text', ''),
+    id: _get(body, 'actions[0].block_id', '')
```

Comment:
oh wow this is the id of the _poll_ isn't it? Not the firebase id for the response itself?  

This confused me, below you do `db.collection("SLACKBOT_TEST").doc(response.id)` and actually get the _poll_ back from firebase. 

I'd suggest changing this to `pollId` or something similar (and then update the `updateDatabase()` function)

Reply:
This IS the firebaseID actually. We were able to find out how to send that when we first send the post to Slack and get it back when Slack post the request to us!